server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs

```diff
@@ -33,8 +33,14 @@ public class AddRequestCommand : Command<PostRequestModel>, IAddRequestCommand
 
         protected override async Task Execute(PostRequestModel parameter, ClaimsPrincipal principal, CancellationToken cancellationToken)
         {
-            var currentYear = DateTime.Now.Year;
+            
+            if (parameter.Amount < 1)
```

Comment:
what about 0.5??

Reply:
Do we have a clear requirement here? If not, I think the 1Euro rule is ok. Nevertheless, error message is misleading.

Reply:
> 
> 
> Do we have a clear requirement here? If not, I think the 1Euro rule is ok. Nevertheless, error message is misleading.

Why misleading ? Its good to be positive :) :)

Reply:
> 
> 
> what about 0.5??

Change to greater than 0